src/main/clojure/clara/rules/durability.clj

```diff
@@ -339,7 +339,8 @@
                         (mem/->RuleOrderedActivation (.-node-id act)
                                                      (index-token internal-seen seen (.-token act))
                                                      (.-activation act)
-                                                     (.-rule-load-order act)))
+                                                     (.-rule-load-order act)
+                                                     false))
```

Comment:
I guess this was broken then?  Going off of my immediate recollection I think this wouldn't matter as long as a session wasn't serialized with activations waiting, which they normally probably aren't (but fixing the bug would still be good).  Sorry, I'll take a closer look within the next few days but a bit pressed for time tonight.  If it is a durability bugfix as well we should update the changelog accordingly.  There might be times when we could have this be true (again, would need to look more closely than I can at the moment) but in any case a value of false should be functionally safe.  Perhaps we should add a test, not necessarily in this PR.

Reply:
I think I need to update the log anyways to reflect that clj-kondo support is baked in anyways, thanks for the reminder. The linter caught that the arity of this call was wrong, I think it sort of works anyways because the field in question is a volatile field and the compiler must be treating it differently.

Reply:
@WilliamParker I'm going to leave this up a bit longer in case you want to review it some more, based on how I understand this works I'm not sure how it ever worked, but somehow it did, I assume the `use-token-identity?` was set to `nil` or `false` by default, but if I try to make this call with only 4 args I get either of these errors:
```
CompilerException java.lang.IllegalArgumentException: No matching ctor found for class clara.rules.memory.RuleOrderedActivation
OR
ArityException Wrong number of args (4) passed to: memory/eval4632/->RuleOrderedActivation--4636
```